int8 sound
int8 stop=-1
int8 startup=0
int8 shutdown=1
int8 arrow_button=2
int8 central_button=3
int8 free_fall=4
int8 collision=5
int8 target_ok=6
int8 target_detect=7